kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -363,6 +363,12 @@ class BatchJobSubmission(
   override def isTimedOut: Boolean = false
 
   override protected def eventEnabled: Boolean = true
+
+  private def cleanupUploadedResourceIfNeeded(): Unit = {
+    if (session.isResourceUploaded && new File(resource).exists()) {
```

Comment:
How about skip check the existence of the file, and then make `Utils.deleteFile` use `Files.deleteIfExists` instead?
```suggestion
    if (session.isResourceUploaded) {
```